NsqSharp/Api/NsqHttpApi.cs

```diff
@@ -154,12 +153,13 @@ protected string GetFullUrl(string route)
 
         /// <summary>POSTs to the specified route using the HTTP address from the constructor.</summary>
         /// <param name="route">The route.</param>
+        /// <param name="contentType">The content type.</param>
         /// <param name="body">The body.</param>
         /// <returns>The response from the server.</returns>
-        protected string Post(string route, byte[] body = null)
+        protected string Post(string route, string contentType = null, byte[] body = null)
```

Comment:
Adding `contentType` in the middle breaks API backwards compatibility in an unnecessary way, suggest moving it to the end if this parameter is needed.

`application/octet-stream` would be more accurate than `application/x-www-form-urlencoded`. Can you use this instead and remove the `contentType` from the user's concern?